src/traces/sankey/render.js

```diff
@@ -155,7 +155,7 @@ function linkModel(uniqueKeys, d, l) {
     var tc = tinycolor(l.color);
     var basicKey = l.source.label + '|' + l.target.label;
     var foundKey = uniqueKeys[basicKey];
-    uniqueKeys[basicKey] = (foundKey === void(0) ? foundKey : 0) + 1;
+    uniqueKeys[basicKey] = (foundKey === void(0) ? 0 : foundKey) + 1;
```

Comment:
@monfera thanks for updating the baseline to :lock: this fix down :tada: 

For documentation purposes, can you explain what's going on this line?